__tests__/app.tests.js

```diff
@@ -74,37 +107,75 @@ describe("/api/articles/:articles_id", () => {
+  test("PATCH: 200 sends the updated article to the client", () => {
+    const vote = { inc_votes: 1 };
     return request(app)
-      .get("/api/articles")
+      .patch("/api/articles/1")
+      .send(vote)
       .expect(200)
       .then((response) => {
-        const { articles } = response.body;
-        expect(articles).toHaveLength(13);
-        articles.forEach((article) => {
-          expect(article).toMatchObject({
-            author: expect.any(String),
-            title: expect.any(String),
-            article_id: expect.any(Number),
-            topic: expect.any(String),
-            created_at: expect.any(String),
-            votes: expect.any(Number),
-            article_img_url: expect.any(String),
-            comment_count: expect.any(String),
-          });
+        const { article } = response.body;
+        expect(article).toMatchObject({
+          article_id: 1,
+          title: expect.any(String),
+          topic: expect.any(String),
+          author: expect.any(String),
+          body: expect.any(String),
+          created_at: expect.any(String),
+          votes: 101,
+          article_img_url: expect.any(String),
         });
       });
   });
-  test("GET 200: returned array is sorted by date in descending order", () => {
+  test("PATCH:404 sends an appropriate status and error message when given a valid but non-existent id", () => {
+    const vote = { inc_votes: 1 };
     return request(app)
-      .get("/api/articles")
-      .expect(200)
+      .patch("/api/articles/9999")
+      .send(vote)
+      .expect(404)
       .then((response) => {
-        const { articles } = response.body;
-        expect(articles).toBeSortedBy("created_at", { descending: true });
+        const { msg } = response.body;
+        expect(msg).toBe("article does not exist");
+      });
+  });
+  test("PATCH:400 sends an appropriate status and error message when given an invalid id", () => {
+    const vote = { inc_votes: 1 };
+    return request(app)
+      .patch("/api/articles/forklift")
+      .send(vote)
+      .expect(400)
+      .then((response) => {
+        expect(response.body.msg).toBe("bad request");
+      });
+  });
+  test("PATCH:400 sends an appropriate status and error when the vote value is 0", () => {
+    const vote = { inc_votes: 0 };
+    return request(app)
+      .patch("/api/articles/1")
+      .send(vote)
+      .expect(400)
+      .then((response) => {
+        expect(response.body.msg).toBe("bad request");
+      });
+  });
+  test("PATCH:400 sends an appropriate status and error message when given an invalid vote", () => {
+    const vote = { inc_votes: "forklift" };
+    return request(app)
+      .patch("/api/articles/1")
+      .send(vote)
+      .expect(400)
+      .then((response) => {
+        expect(response.body.msg).toBe("bad request");
+      });
+  });
+  test("PATCH:400 sends an appropriate status and error message when the request is more than a single vote", () => {
```

Comment:
This should also be acceptable, I would actually recommend adding another couple of Status 200 tests for if inc_votes is 0, inc_votes is a negative number and inc_votes is a larger number than 1